src/openrct2/audio/audio.h

```diff
@@ -170,6 +161,22 @@ extern rct_ride_music_params* gRideMusicParamsListEnd;
 
 extern rct_vehicle_sound gVehicleSoundList[AUDIO_MAX_VEHICLE_SOUNDS];
 
+/**
+ * Returns false when no audio device is available or when audio is turned off, otherwise true.
+ */
+bool audio_is_available();
+/*
+ * Returns the amount of available audio devices.
+ */
+int32_t audio_get_device_count();
+/**
+ * Returns the device name by index.
+ */
+const std::string& audio_get_device_name(int32_t index);
+/**
+ * Returns the currently used device index, -1 if not available.
+ */
+int32_t audio_get_device_index();
```

Comment:
Shouldn't we be taking this opportunity to use `TitleCase` in the new functions?

Reply:
I want to move all of those into a class, so better left for a new PR. Just wanted to make it easy to move.